.github/workflows/tests.yml

```diff
@@ -44,12 +44,16 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -e .[testing,pdf_html]
+        pip install -e .[testing,pdfhtml]
 
     - name: Install Headless Chrome dependencies
       run: |
         sudo apt-get install -yq $(cat .github/workflows/pyppeteer_reqs.txt)
 
+    - name: Install texlive
+      run: |
+        bash scripts/install_latex.sh
+        export PATH="/tmp/texlive/bin/x86_64-linux:$PATH"
```

Comment:
@choldgraf do you know how the update the environmental variable PATH in Github Actions?

Reply:
hmm for this I'm not as sure, I'd look around https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables